interface/settings.py

```diff
@@ -51,10 +51,12 @@
 AUTH_LDAP_CACHE_TIMEOUT = 3600
 
 AUTHENTICATION_BACKENDS = [
-    'django_auth_ldap.backend.LDAPBackend',
     "django.contrib.auth.backends.ModelBackend",
 ]
 
+if os.environ.get('LDAP_SERVER_URI'):
+	AUTHENTICATION_BACKENDS.insert(0, 'django_auth_ldap.backend.LDAPBackend')
```

Comment:
It looks like flake8 really doesn't like this line:

https://jenkins.liquiddemo.org/vmck/acs-interface/753/1/2